### ISSUE TITLE
Affichage des événements sous forme de calendrier et de liste

### ISSUE THREAD

**Issue by srosset81** [chemins-de-la-transition/semapps#45]
Pour l'affichage en calendrier, nous avons une issue ici: https://github.com/assemblee-virtuelle/semapps/issues/397

Pour l'affichage sous forme de liste, il faudrait un petit travail pour faire quelque chose de joli, possiblement à la Gogocarto:

![Capture d’écran de 2021-04-02 17-35-08](https://user-images.githubusercontent.com/17931931/113430082-d8d5ab00-93d9-11eb-94ec-78f9aabf64d0.png)

Voir https://test-event.gogocarto.fr/annuaire#/liste/@48.635,4.929,14z?cat=all

L'affichage sous forme de liste serait un peu limitant lorsqu'on a des événements qui durent plusieurs jours et qui se chevauchent. En revanche ça serait adapté pour afficher sur une page parcours, où les événements s'enchaînent.

**Comment by GuillaumeAV** [comment]
Yes ! En effet le design style gogocarto est vraiment classe ! Si pas couteux ca pourrait être top et je vote pour ! 
Concernant la vue calendrier on peut différer ?
